### PR TITLE
Add api function (get group's allprojects)

### PR DESCRIPTION
**Pull request by utolosa002**

### CODE REVIEW

**Comment by utolosa002** [comment]
@m1guelpf done.

**Comment by GrahamCampbell** [comment]
Friendly ping @utolosa002. :)

File: lib/Gitlab/Api/Groups.php [hunk 1]
```diff
@@ -198,7 +198,25 @@ public function projects($id, array $parameters = [])
     }
 
     /**
-     * @param int $groupId
+     * @param $id
```
Comment (GrahamCampbell) [inline lib/Gitlab/Api/Groups.php:201]: invalid syntax

File: .travis.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ php:
   - 7.0
   - 7.1
   - 7.2
+  - 7.3
```
Comment (GrahamCampbell) [inline .travis.yml:10]: Please rebase the PR.

File: lib/Gitlab/Api/Groups.php (hunk 1 above)
Comment (utolosa002) [reply lib/Gitlab/Api/Groups.php]: fixed: https://github.com/utolosa002/php-gitlab-api/commit/f145bb1976e5a2081823e44714f829e67d82ef4b

File: .travis.yml (hunk 2 above)
Comment (utolosa002) [reply .travis.yml]: fixed: https://github.com/utolosa002/php-gitlab-api/commit/068e0f43ce3fadb2778e28dd3bc7f5b51e5a2df1

File: lib/Gitlab/Api/Groups.php [hunk 3]
```diff
@@ -198,7 +198,25 @@ public function projects($id, array $parameters = [])
     }
 
     /**
-     * @param int $groupId
+     * @param int   $group_id
+     *
+     * @return mixed
+     */
+    public function allprojects($group_id)
```
Comment (GrahamCampbell) [inline lib/Gitlab/Api/Groups.php:205]: Please name this method `allProjects`

**Comment by GrahamCampbell** [comment]
Oh, actually, I see what this does now. In order to fetch all responses, you must use the `ResultPager`. No changes are needed to this class.

**Comment by GrahamCampbell** [comment]
Please see the example on the readme: https://github.com/m4tthumphrey/php-gitlab-api#example-with-pager.

**Pull request closed without merging**